java/arcs/core/data/expression/Evaluator.kt

```diff
@@ -184,6 +184,12 @@ sealed class GlobalFunction(val name: String) {
             toSequence<Int>(args[0]).average()
     }
 
+    /** Find the average of a [Sequence]. */
+    object Sum : GlobalFunction("sum") {
+        override fun invoke(evaluator: ExpressionEvaluator, args: List<Any?>) =
+            toSequence<Int>(args[0]).sum()
+    }
+
```

Comment:
Neeed to register it in the companion object field below (line 208), otherwise the parser can't look it up.

Reply:
Ah, thanks. Done.